theia-cpp-docker/README.md

```diff
@@ -1,16 +1,19 @@
-## Theia cpp Docker
+# Theia cpp Docker
```

Comment:
I was thinking, should we update the readme title to **Theia C/C++ Docker** so it's clearer?

Reply:
Just that entry or the image name as well? The next line mentions "C/C++" so I think it's not too confusing?

Reply:
I was thinking just the title but I'm fine either way (and it's not related to your changes).